lua/mason-nvim-dap/mappings/configurations.lua

```diff
@@ -263,4 +263,37 @@ M.haskell = {
 	},
 }
 
+M['pwa-node'] = {
```

Comment:
this seem like sensible defaults. where did you get them from?

Reply:
I had these configs running with https://github.com/mxsdev/nvim-dap-vscode-js/blob/03bd29672d7fab5e515fc8469b7d07cc5994bbf6/README.md?plain=1#L79. They were later updated based on https://github.com/microsoft/vscode-js-debug/blob/2152210dc7c3933e2b4ef7c72d72cf2fef765760/src/dap/api.d.ts#L229